### PR TITLE
Update USING_PRO.md

### PR DESCRIPTION
**Pull request by linuxbckp**
Add document about #3487

### CODE REVIEW

**Comment by linuxbckp** [comment]
OK. I changed to Marked part.

File: docs/USING_ADVANCED.md [hunk 1]
```diff
@@ -13,6 +13,31 @@ const marked = new Marked([options, extension, ...]);
 |:--------|:-------|:----------------------------------------------------------------------|
 | options |`object`|The same arguments that can be passed to [`marked.use`](/using_pro#use)|
 
+Another case is, if you want to rerun function everytime they open a file, or you want to make marked.run() non-state, you can create a new instance of Marked to ensure options and extensions are locally scoped.
```
Comment (UziTech) [inline docs/USING_ADVANCED.md:16]: It doesn't seem like this is saying anything more than the documentation above. That the user should use Marked instance if they want extensions to be locally scoped.

I think the example should emphasize that `marked.use(...)` should not be used in a loop or function. It should only be used directly after `new Marked` is created or `marked` is imported

**Review by UziTech — CHANGES_REQUESTED** [review]

**Comment by linuxbckp** [comment]
OK. I changed to a note.

**Review by UziTech — APPROVED** [review]
Thanks! 💯

**Pull request merged**